Makefile

```
@@ -72,11 +75,13 @@ $(XCPROJECT): $(PROJECT)/project.yml
 $(PROJECT)/VPNLib/vpn.pb.swift: $(PROJECT)/VPNLib/vpn.proto
 	protoc --swift_opt=Visibility=public --swift_out=. 'Coder-Desktop/VPNLib/vpn.proto'
 
-$(PROJECT)/VPNLib/FileSync/daemon.pb.swift: $(PROJECT)/VPNLib/FileSync/daemon.proto
+# All other protobuf targets are vendored from Mutagen
+%.pb.swift:
```

The target of this make recipe is not constrained to the MutagenSDK dir but seems to only function in that dir with the protoc flags you used.

Try:
```suggestion
$(MUTAGEN_PROTO_SWIFTS): %.pb.swift:
```